src/pybind/mgr/telemetry/module.py

```diff
@@ -656,7 +711,7 @@ def get_osd_histograms(self, mode: str = 'separated') -> List[Dict[str, dict]]:
                 # schema when it doesn't. In either case, we'll handle that
                 # by continuing and collecting what we can from other osds.
                 except (json.decoder.JSONDecodeError, KeyError) as e:
-                    self.log.debug("Error caught on osd.{}: {}".format(osd_id, e))
+                    self.log.error("Error caught on osd.{}: {}".format(osd_id, e))
```

Comment:
```suggestion
                    self.log.exception("Error caught on osd.{}: {}".format(osd_id, e))
```

Reply:
@ljflores the `self.log.exception` fixes were lost in the force push in this main PR and its backport; not a big issue, can be fixed later